.eslintrc.js

```diff
@@ -57,8 +58,15 @@ module.exports = {
     "node/no-unpublished-require": "off",
     "node/no-unsupported-features/es-syntax": "off",
 
+    "notice/notice":[ "error", { mustMatch: "This Source Code Form is subject to the terms of the Mozilla Public", "templateFile": "copyright.txt" } ],
+
+    "eol-last": "warn",
+    "eqeqeq": "off", // TODO: "warn",
```

Comment:
Why do we need this?

Reply:
This is just a good rule in general since `===` is pretty rad.
But I don't have super strong feelings about many of these rules, so I can remove them for now or feel free to delete and merge if that's easier (since its midnight:48 locally and I probably won't be online much longer).